kd/deployment.yaml

```
@@ -0,0 +1,255 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: hocs-camunda-tools
+  labels:
+    version: {{.VERSION}}
+  annotations:
+    downscaler/uptime: {{.UPTIME_PERIOD}}
+    #downscaler/uptime: "Mon-Fri 08:00-18:00 Europe/London"
+spec:
+  replicas: {{.MIN_REPLICAS}}
+  selector:
+    matchLabels:
+      name: hocs-camunda-tools
+  strategy:
+    rollingUpdate:
+      maxUnavailable: 0
+      maxSurge: 2
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        name: hocs-camunda-tools
+        role: hocs-camunda-tools
+        version: {{.VERSION}}
+    spec:
+      containers:
+      - name: certs
+        image: quay.io/ukhomeofficedigital/cfssl-sidekick-jks:v0.0.6
+        securityContext:
+          runAsNonRoot: true
+          capabilities:
+            drop:
+             - SETUID
+             - SETGID
+        args:
+          - --certs=/certs
+          - --domain=hocs-camunda-tools.{{.KUBE_NAMESPACE}}.svc.cluster.local
+          - --expiry=8760h
+          - --command=/usr/local/scripts/trigger_nginx_reload.sh
+        env:
+          - name: KUBE_NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
+        volumeMounts:
+          - name: certs
+            mountPath: /certs
+          - name: bundle
+            mountPath: /etc/ssl/certs
+            readOnly: true
+        resources:
+          limits:
+            memory: 64Mi
+            cpu: 900m
+          requests:
+            memory: 16Mi
+            cpu: 300m
+
+      - name: proxy
+        image: quay.io/ukhomeofficedigital/nginx-proxy-govuk:v4
+        imagePullPolicy: Always
+        securityContext:
+          runAsNonRoot: true
+          capabilities:
+            drop:
+             - SETUID
+             - SETGID
+        env:
+          - name: PROXY_SERVICE_HOST
+            value: '127.0.0.1'
+          - name: PROXY_SERVICE_PORT
+            value: '8081'
+          - name: LOCATIONS_CSV
+            value: '/, /api/'
+          - name: NAXSI_USE_DEFAULT_RULES
+            value: 'FALSE'
+          - name: ENABLE_UUID_PARAM
+            value: 'FALSE'
+          - name: HTTPS_REDIRECT
+            value: 'TRUE'
+          - name: SERVER_CERT
+            value: /certs/tls.pem
+          - name: SERVER_KEY
+            value: /certs/tls-key.pem
+          - name: ERROR_REDIRECT_CODES
+            value: '501 502 503 504'
+          - name: ADD_NGINX_HTTP_CFG
+            value: 'fastcgi_buffers 16 64k; fastcgi_buffer_size 128k; proxy_buffer_size 128k; proxy_buffers 4 64k; proxy_busy_buffers_size 128k; client_header_buffer_size 8k; large_client_header_buffers 4 128k;'
```

You can use (https://yaml-multiline.info/)[YAML multiline strings] to make this easier to read:
```suggestion
            value: >
              fastcgi_buffers 16 64k;
              fastcgi_buffer_size 128k;
              proxy_buffer_size 128k;
              oroxy_buffers 4 64k;
              proxy_busy_buffers_size 128k;
              client_header_buffer_size 8k;
              large_client_header_buffers 4 128k;
```